docs/user/ai_model_serving.md

```diff
@@ -0,0 +1,517 @@
+# AI Model Serving on MicroShift
+
+AI Model Serving on MicroShift is a single-model serving platform for AI models.
+It includes limited subset of Red Hat OpenShift AI (RHOAI):
+[KServe in Raw Deployment Mode](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.17/html/serving_models/serving-large-models_serving-large-models#raw_deployment_mode)
+and the `ServingRuntimes` object type that is namespace-specific.
+Now you can train your models in the cloud and serve them on the edge.
+
+## Definitions
+
+- [ServingRuntime](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.17/html/serving_models/serving-large-models_serving-large-models#servingruntime)
+  - For more information about Serving Runtimes refer to [upstream Kserve documentation](https://kserve.github.io/website/latest/modelserving/servingruntimes/)
+- [InferenceService](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.17/html/serving_models/serving-large-models_serving-large-models#inferenceservice)
+
+## Supported model-serving runtimes
+
+Currently MicroShift supports following model-serving runtimes:
+- OpenVINO Model Server
+- vLLM ServingRuntime for KServe
+
+Following runtimes are included but not supported at the time:
+- Caikit Text Generation Inference Server (Caikit-TGIS) ServingRuntime for KServe
+- Caikit Standalone ServingRuntime for KServe
+- Text Generation Inference Server (TGIS) Standalone ServingRuntime for KServe
+- vLLM ServingRuntime with Gaudi accelerators support for KServe
+- vLLM ROCm ServingRuntime for KServe
+
+Refer to
+[RHOAI documentation](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.17/html/serving_models/serving-large-models_serving-large-models#supported-model-serving-runtimes_serving-large-models).
+for details about the model-serving runtimes.
+
+## General usage overview
+
+1. Develop, train, test, and prepare model for serving
```

Comment:
Should these be links to the appropriate chapters in the doc, or there's no one-to-one mapping?
Also, why do we need Optional sections?

Reply:
For example, that first point doesn't have a section. It's more of a prerequisite.
Also, the list is an high level step by step to give idea about the order of actions, not really a ToC

Reply:
Yes, optional may be confusing, especially in the overview. We should attempt simplifying as much as we can.